src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+
+  if (days < 3) {
+    return sum;
+  } else if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  } else if (days < 7) {
```
you already check that days >= 3 in previous if